glide.yaml

```diff
@@ -24,6 +23,14 @@ import:
   version: ^1.0.0
   subpackages:
   - proto
+- package: github.com/couchbase/vellum
+  version: eb6ae3743b3f300f2136f83ca78c08cc071edbd4
+- package: github.com/edsrzf/mmap-go # un-used but required for a compile time dep from vellum
+  version: 0bce6a6887123b67a60366d2c9fe2dfb74289d2e
+- package: github.com/pilosa/pilosa
```

Comment:
Might it be worth specifying that you're only pulling in `github.com/pilosa/pilosa/roaring`?

Reply:
hm doesn't the sub-package specifier below this indicate that?